app/src/main/java/tech/relaycorp/gateway/data/preference/PublicGatewayPreferences.kt

```diff
@@ -56,6 +58,7 @@ class PublicGatewayPreferences
 
     suspend fun setCertificate(value: Certificate) {
         certificate.setAndCommit(Base64.encodeToString(value.serialize(), Base64.DEFAULT))
```

Comment:
I think it'd be best not to stop the certificate any more now that we're integrating this change. Instead of storing the certificate, I think we should only store its `subjectPrivateAddress`.

The original version of this app stored the public gateway's certificate because that was also used for encryption. However, now that the Channel Session Protocol is integrated, private gateways need not store their public gateways' certificates.

(The public gateway will always include its certificate when registering a private gateway, but that's mainly so that the private gateway can log or store specific elements of the certificate, and so that the private gateway can verify that its new certificate was issued by the public gateway)